pkg/stdlib/html/driver/dynamic/document.go

```diff
@@ -753,3 +753,57 @@ func (doc *HTMLDocument) Navigate(url values.String) error {
 
 	return doc.WaitForNavigation(5000)
 }
+
+type ScreenshotArgs struct {
+	X       float64
+	Y       float64
+	Width   float64
+	Height  float64
+	Format  string
```

Comment:
Can we create an enum type instead of the magic string? It will improve the readability.
Something like this (you can come up with a better naming):

```go

type ScreenshotFormat string

const (
  ScreenshotFormatPNG ScreenshotFormat = "png"
  ScreenshotFormatJPEG ScreenshotFormat = "jpeg"
)

```